tests/remote_attestation/maa/oe_enclave/enc/public.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This is the matching public.pem which is used for private.pem for signing oe_enclave app. I also didn't find code  using it directly. Should we keep it just in case.

Reply:
When in doubt, remove unnecessary key files, don't keep them "just in case", you can easily generate new key pairs.

Reply:
Verified that this file is not needed. Removed it.